### ISSUE TITLE
boost-presets not working (just the single boost) for hot-water

### ISSUE THREAD

**Issue by elden1337** [elupus/hass_nibe#175]
When using latest hass_nibe (1.5.7) the water-heater operation preset is not working for boost-modes other than boost_one_time. 
Got an F730 if that helps but assume this is a generic api-missing-feature as of now.

```
Logger: homeassistant.components.websocket_api.http.connection
Source: custom_components/nibe/water_heater.py:192
Integration: Home Assistant WebSocket API (documentation, issues)
First occurred: 12:47:25 (1 occurrences)
Last logged: 12:47:25

[547314247680] Operation mode boost_three_hours not supported in nibe api
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/components/websocket_api/commands.py", line 199, in handle_call_service
    await hass.services.async_call(
  File "/usr/src/homeassistant/homeassistant/core.py", line 1820, in async_call
    task.result()
  File "/usr/src/homeassistant/homeassistant/core.py", line 1857, in _execute_service
    await cast(Callable[[ServiceCall], Awaitable[None]], handler.job.target)(
  File "/usr/src/homeassistant/homeassistant/helpers/entity_component.py", line 216, in handle_service
    await service.entity_service_call(
  File "/usr/src/homeassistant/homeassistant/helpers/service.py", line 798, in entity_service_call
    future.result()  # pop exception if have
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 977, in async_request_call
    await coro
  File "/usr/src/homeassistant/homeassistant/helpers/service.py", line 838, in _handle_entity_call
    await result
  File "/config/custom_components/nibe/water_heater.py", line 192, in async_set_operation_mode
    raise Exception(
Exception: Operation mode boost_three_hours not supported in nibe api

```

**Comment by elden1337** [comment]
Ok, after the error message it still pushed and started a three-hour boost. So guess an internal bug.

**Comment by Larrax** [comment]
I have the same error with F470. HA also seems to have problems acquiring the current operating mode, as it always shows as "off" even when the "Hot water" mode is running. Sometimes, I also noticed the command did not turn on the heating, even though this might be because of the extended response times - when I experimented with it too quickly.

![image](https://github.com/elupus/hass_nibe/assets/33988353/b09b3800-9a66-4a7a-a6e4-3e05f1e6fd4a)
